<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Signup</class>
 <widget class="QWidget" name="Signup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1340</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_image">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-5</y>
     <width>1361</width>
     <height>861</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/background/register.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2_back">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>461</width>
     <height>641</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:18px;
background-color: rgb(255, 255, 255);
border-top: 5px solid qlineargradient(y0:0, y1:1,stop: 0 #ececef, stop: 1 white);
border-left: 5px solid qlineargradient(x0:0, x1:1,stop: 0 #ececef, stop: 1 white);
 border-bottom: 5px solid qlineargradient(y0:0, y1:1,stop: 0 white, stop: 1  #ececef);
border-right: 5px solid qlineargradient(x0:0, x1:1,stop:  0 white, stop: 1 #ececef);


</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;SourceHanSansSC&quot;;
color: rgba(0,0,0,1);</string>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_id1">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>270</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Candara</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(247, 247, 247);
border:1px groove gray;border-radius:
7px;padding:2px 4px;
font: 12pt &quot;Candara&quot;;

</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Please enter your username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_id1_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>370</y>
     <width>311</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Candara</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(247, 247, 247);
border:1px groove gray;border-radius:
7px;padding:2px 4px;
font: 12pt &quot;Candara&quot;;

</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Please input a password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>650</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0.52, y1:1, x2:0.54, y2:0, 
stop:0.0112994 rgba(110,102,162,1), 
stop:1 rgba(110,102,162,1));
color: rgb(255, 255, 255, 255);
 
border:0px groove gray;border-radius:
15px;padding:2px 4px;
font: 18pt &quot;Candara&quot;;

</string>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_id1_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>460</y>
     <width>311</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Candara</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(247, 247, 247);
border:1px groove gray;border-radius:
7px;padding:2px 4px;
font: 12pt &quot;Candara&quot;;

</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="placeholderText">
    <string>Please input a mail</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_id1_4">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>550</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Candara</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(247, 247, 247);
border:1px groove gray;border-radius:
7px;padding:2px 4px;
font: 12pt &quot;Candara&quot;;

</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Verification Code</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Cancel_B">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>650</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0.52, y1:1, x2:0.54, y2:0, 
stop:0.0112994 rgba(110,102,162,1), 
stop:1 rgba(110,102,162,1));
color: rgb(255, 255, 255, 255);
background-color: rgb(219, 210, 216);
 
border:0px groove gray;border-radius:
15px;padding:2px 4px;
font: 18pt &quot;Candara&quot;;

</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Cancel_B_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>550</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0.52, y1:1, x2:0.54, y2:0, 
stop:0.0112994 rgba(110,102,162,1), 
stop:1 rgba(110,102,162,1));
color: rgb(255, 255, 255, 255);
background-color: rgb(78, 62, 158);
 
border:0px groove gray;border-radius:
10px;padding:2px 4px;
font: 18pt &quot;Candara&quot;;

</string>
   </property>
   <property name="text">
    <string>Gain</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
